packages/column-views/src/maps/column-navigation/mapbox/column-navigation-map.stories.ts

```diff
@@ -25,6 +25,7 @@ export default {
 export const ColumnNavigation = {
```

Comment:
Rather than updating this story, it'd be better to create a new one. That way you can ensure that the current behavior doesn't change when you add the new filtering capability.